src/Sylius/Bundle/AdminApiBundle/Resources/config/grids/product_review.yml

```diff
@@ -0,0 +1,46 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+#
+# @author Paul Stoica <paul.stoica18@gmail.com>
```

Comment:
We are not using author docblocks anymore, because they are just redundant (data can be fetched from git history), please remove it from here and other files